tests/data/cromwell_options.json

```diff
@@ -0,0 +1,4 @@
+{
+    "final_workflow_outputs_dir": "test-output",
+    "use_relative_output_paths": true
+}
```

Comment:
@DavyCats @rhpvorderman Do we still specify "default_runtime_attributes" here? Or is that not explicitly required?

Reply:
Good one, this file should look like:
```
{
"final_workflow_outputs_dir": "test-output",
"use_relative_output_paths": true,
"default_runtime_attributes": {
  "docker_user": "$EUID"
  }
}
```